src/mixins/date-mixin.ts

```diff
@@ -99,6 +113,30 @@ export default class DateMixin extends Mixins(CommonMixin) {
     return dateStr
   }
 
+  /**
+   * Converts a date string of YYYY-mm-dd format to a date string with more display options in Pacific timezone.
+   * @example "2021-01-01" -> "Dec 31, 2020"
+   */
+  dateStrToPacificDate (dateStr: string, longMonth = false, showWeekday = false): string {
+    // safety checks
+    if (!dateStr) return null
+    const date = this.createUtcDateFromStr(dateStr)
```

Comment:
I'm glad you copied this from another project, but please make it exactly the same, so that it can be moved to the shared components repo at some point.

Reply:
I'm not sure what you exactly mean by copied from another project.  Most of the functions in this file were already in place.  I had specific use case which is why I needed to create this function.  Although I realize now that I can simplify this function a lot by calling `dateToPacificDate`.

Reply:
Sorry, I didn't realize you cloned an existing local method. No offense intended!